.gitignore

```diff
@@ -18,6 +18,7 @@ pids
 .env.*
 
 # Editor directories and files
+.idea

```

Comment:
In this repo we have https://github.com/genesiscommunitysuccess/blank-app-seed/tree/main/.idea which we intentionally keep in version control. We can exclude subfolders of .idea if needed